bouncer/src/repo/core/model/bson/repo_bson_builder.h

```diff
@@ -145,6 +145,14 @@ namespace repo {
 					mongo::BSONObjBuilder::append(label, date);
 				}
 
+				/**
+				* Appends a Vector but as an object, instead of an array.
+				*/
+				void appendVector3d(
```

Comment:
would we maybe call it `appendVector3DObject`? 

just to be clear (as whoever is coding might miss the comment, esp if they're using intellisense!)